docs/development.md

```diff
@@ -60,3 +59,50 @@ After following the above steps, you should now be able to start contributing to
 this integration. The integration will pull in the `CLIENT_ACCESS_TOKEN` and
 `CLIENT_NAMESPACE` variables from the `.env` file and use them when making
 requests.
+
+## Testing API Calls
```

Comment:
I love to see these. What are your thoughts on pushing this as an actual script in a sep file like I did for qualys? Or do you see more value in posting like this?

https://github.com/JupiterOne/graph-qualys/blob/main/docs/jupiterone.md#troubleshooting-qualys-user-credentials

Reply:
Ah, good idea. I should have done that. 👍